src/main/resources/model/instance.json

```diff
@@ -47,7 +47,7 @@
           "index": "keyword"
         },
         "identifierTypeId": {
-          "index": "none"
+          "index": "source"
```

Comment:
this change is not relevant to the expandAll functionality, is this preparation for some story?

Reply:
I needed it for testing to be able to use equals assertion, but this identifier should be also required for ISBN normalized stories, so I just decided to add it to index.